src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5171,4 +5171,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureRefExtensionMethods" xml:space="preserve">
     <value>ref or in extension methods</value>
   </data>
+  <data name="ERR_StackAllocConversionNotPossible" xml:space="preserve">
+    <value>Conversion of a stackalloc expression of type '{0}' to type '{1}' is not possible.</value>
+  </data>
+  <data name="IDS_StackAllocExpression" xml:space="preserve">
+    <value>stackalloc expression of type '{0}'</value>
```

Comment:
This is used as a fill-in for other diagnostics. I don't think this text will work out very well for that purpose. I recommend
    <value>'stackalloc {0}[{1}]'</value>
and pass in the element type and count when formatting it